src/main/java/org/hawkular/client/android/fragment/AlertsFragment.java

```diff
@@ -207,9 +207,10 @@ private void setUpAlertsTriggers(List<Trigger> triggers) {
         List<Trigger> filteredTriggers = new ArrayList<>();
 
         for (Trigger trigger : triggers) {
-            if (trigger.getId().endsWith(resource.getId())) {
-                filteredTriggers.add(trigger);
-            }
+                if ( trigger.getTags()!=null && trigger.getTags().containsKey("resourceId") && trigger.getTags().get
```

Comment:
The first part you are dealing on the ` != null`. The second part (may or not contain resourceId) is dealt with the equals on the `resource.getId()`: if `getTags().get("resourceId")` returns null, it won't be equal to `resource.getId()`.